### ISSUE TITLE
[v5]If I delete all projects I can't login anymore

### ISSUE THREAD

**Issue by b1zzu** [reportportal/reportportal#838]
**Describe the bug**

I deleted all Projects using the `superadmin` user and then I couldn't login anymore

```
An error occurred while connecting to server: You do not have enough permissions. Project role 'null' not found. Did you use correct Role Name?
```

**To Reproduce**

**Expected behavior**

To be able to login or somehow recreate a project using the ui.

**Desktop:**
 - OS: Fedora
 - Browser chrome

**Comment by DzmitryHumianiuk** [comment]
Well, i would say it's expected :)
no projects - no permission to see anything.

but we definitely should think of natural way to process it.

**Comment by b1zzu** [comment]
Hi, I understand that's a strange situation, but having at least the possibility to create a new project if there are no one will be nice

**Comment by DzmitryHumianiuk** [comment]
@b1zzu there should be a mechanism to keep personal project always available.
Login to RP, where there is no any project, should create personal space for user (personal project).

if it is not - than it is a bug.

**Comment by b1zzu** [comment]
@DzmitryHumianiuk maybe it's not doing it for the admin user, or the mechanism is in place only for the user accessing using external authentication (github, sso, ..)

**Comment by DzmitryHumianiuk** [comment]
@b1zzu looks like for external authes only.
@e-olga @evjlobanova please double check and consider as a bug if occurs.

**Comment by e-olga** [comment]
Good day,
@b1zzu, @DzmitryHumianiuk , the issue is fixed in version 5.1.

Regards,
Olga
